requirements.txt

```diff
@@ -9,8 +9,9 @@ black==23.1.0
 aiosqlite==0.18.0
 databases==0.7.0
 SQLAlchemy==1.4.46
-asyncpg==0.27.0
-psycopg2-binary==2.9.5
+asyncpg==0.29.0
+psycopg2-binary==2.9.9
 alembic==1.10.2
 geopy==2.3.0
-pytest-dependency==0.5.1
\ No newline at end of file
+pytest-dependency==0.5.1
+pydantic==2.9.1
```

Comment:
I missed this one, I think there's no need to require pydantic explicitly, since fastapi should pull the right version...

Reply:
That's because on my arm Mac (don't know which one is the problem) pulls a different version with different errors, so locking in the newest version for now helps getting a consistent set of issues.

I guess that when all the remaining set of dependencies + python can be upgraded some of these will go away.